### PR TITLE
Cloned channels no auto #1211047 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/22461
https://github.com/uyuni-project/uyuni-docs/issues/2280
https://bugzilla.suse.com/show_bug.cgi?id=1211047

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
@wweellddeerr , I'd appreciate if you'd help reviewing this.  Is this added at the right place?  Or would users better see is somewhere else?

If it helps, I can create and add rendered output.

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

**Comment by keichwa** [comment]
> Actually, I would like to have this recommendations in [Software Channels section](https://documentation.suse.com/suma/4.3/en/suse-manager/reference/systems/system-details/sd-software-channels.html#_base_channel). There we state the following:
> 
> > Mandatory child channels are pre-selected.
>
> I would like to make it clear that this pre-selection is necessary to avoid problems [...]

Thanks, @wweellddeerr , this info now is basically!  Now I'd like to ask @admd whether it can stay that way, or whether changes are needed to make users understand what this feature is about.  Abid, can you please take a look if time permits? Thanks to all for all your help and feedback!

File: modules/reference/pages/systems/system-details/sd-software-channels.adoc [hunk 1]
```diff
@@ -20,6 +19,22 @@ The child channels associated with the base channel are populated in the [guimen
...
+By default, cloned vendor channels match the original vendor channel and automatically select the dependencies.
+You can disable the automatic selection for cloned channels by adding the following option in [path]``/etc/rhn/rhn.conf``:
+----
+java.cloned_channel_auto_selection = false
+----
+====
+
+In general, channel pre-selection is necessary to avoid problems with missing dependencies for a given channel but it only works flawlessly for vendor channels.
+For cloned channels, {productname} relies on the channel label to define dependencies and use it in auto selection.
+This heuristic sometimes causes a side-effect of having unnecessary channels being automatically selected as dependencies.
```
Comment (admd) [inline modules/reference/pages/systems/system-details/sd-software-channels.adoc:35]:
```suggestion
This heuristic sometimes lead to either selecting unnecessary child channels or overlooking some essential ones. Users should exercise caution and pay attention in such case that correct channels are selected.
```

File: modules/reference/pages/systems/system-details/sd-software-channels.adoc [hunk 2]
```diff
@@ -20,6 +19,22 @@ The child channels associated with the base channel are populated in the [guimen
...
+In such a case, it can be necessary to disable auto selection using the above mentioned setting.
```
Comment (admd) [inline modules/reference/pages/systems/system-details/sd-software-channels.adoc:36]: I will remove this line as we have already covered it in note.

**Review by admd — COMMENTED** [review]
just small nitpicks but looks good overall.

File: modules/reference/pages/systems/system-details/sd-software-channels.adoc (hunk 2 above)
Comment (keichwa) [reply modules/reference/pages/systems/system-details/sd-software-channels.adoc]: yes, makes sense!

**Pull request merged**